### DISCUSSION TITLE
Onboarding

### DISCUSSION THREAD

**Discussion by jananiravi** [JRaviLab/MolEvolvR#25] [Announcements]
- [ ] contribution guidelines
- [ ] onboarding doc, incl. installation
- [ ] style guide
- [ ] #7 
- [ ] MolEvolvR -- rename throughout and add phonetics upfront
- [ ] glossary, as needed

**Comment by Ajoke23** [comment]
Hi @jananiravi. I am interested in this project. I am an Outreachy applicant
